tests/foreman/api/test_errata.py

```diff
@@ -617,14 +621,15 @@ def test_positive_get_diff_for_cv_envs():
 
 @pytest.mark.skip_if_open("BZ:2013093")
```

Comment:
Non-blocking comments:
The BZ seems to be closed you can remove the `@pytest.mark.skip_if_open("BZ:2013093")`